includes/admin/views/builder/video-explainer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Builder video explainer view
+ *
+ * @since [version]
+ * @version [version]
+ */
+?>
+<script type="text/html" id="tmpl-llms-video-explainer-template">
+
+	<span class="llms-video-explainer-trigger">
+		<img
+			src="<?php echo plugin_dir_url( LLMS_PLUGIN_FILE ); ?>assets/images/course-builder-video-thumbnail.jpg"
```

Comment:
```suggestion
			src="<?php echo esc_url( plugin_dir_url( LLMS_PLUGIN_FILE ) . 'assets/images/course-builder-video-thumbnail.jpg' ); ?>"
```
let's escape the url.

Reply:
fc96051